torch/testing/_internal/common_modules.py

```diff
@@ -415,7 +416,11 @@ def module_inputs_torch_nn_Embedding(module_info, device, dtype, requires_grad,
     ModuleInfo(torch.nn.Linear,
                module_inputs_func=module_inputs_torch_nn_Linear),
     ModuleInfo(torch.nn.Bilinear,
-               module_inputs_func=module_inputs_torch_nn_Bilinear),
+               module_inputs_func=module_inputs_torch_nn_Bilinear,
+               decorators=(
+                    DecorateInfo(
```

Comment:
minor lint issue here

Also could you please specify the tolerance override only for the specific test it's failing on?

Reply:
done!